client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/price-summary.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Tooltip from 'components/tooltip';
+import formatCurrency from 'lib/format-currency';
+import {
+	getTotalPriceBreakdown,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+
+class PriceSummary extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			tooltipVisible: false,
+		};
+	}
+
+	showTooltip = () => {
+		this.setState( { tooltipVisible: true } );
+	};
+
+	hideTooltip = () => {
+		this.setState( { tooltipVisible: false } );
+	};
+
+	setTooltipContext = tooltipContext => {
+		if ( tooltipContext ) {
+			this.setState( { tooltipContext } );
+		}
+	};
+
+	renderRow = ( itemName, itemCost, key, isTotal, isDiscount ) => {
+		const { translate } = this.props;
+		const className = classNames( 'label-purchase-modal__price-item', {
+			'label-purchase-modal__price-item-total': isTotal,
+		} );
+		return (
+			<div key={ key } className={ className }>
+				<div className="label-purchase-modal__price-item-name">
+					{ itemName }
+				</div>
+				{ isDiscount &&
+					<div className="label-purchase-modal__price-item-help">
+						<Gridicon
+							ref={ this.setTooltipContext }
+							icon="help-outline"
+							onMouseEnter={ this.showTooltip }
+							onMouseLeave={ this.hideTooltip }
+							size={ 18 } />
+						<Tooltip
+							className="label-purchase-modal__price-item-tooltip is-dialog-visible"
+							isVisible={ this.state.tooltipVisible }
+							context={ this.state.tooltipContext } >
+						{ translate( 'WooCommerce Services gives you access to USPS ' +
+							'Commercial Pricing, which is discounted over Retail rates.' ) }
+						</Tooltip>
+					</div> }
+				<div className="label-purchase-modal__price-item-amount">{ formatCurrency( itemCost, 'USD' ) }</div>
+			</div>
+		);
+	};
+
+	render() {
+		const { priceBreakdown, translate } = this.props;
+		if ( ! priceBreakdown ) {
+			return null;
+		}
+
+		const { prices, discount, total } = priceBreakdown;
+
+		return (
+			<Card>
+				{ prices.map( ( service, index ) => this.renderRow( service.title, service.retailRate, index ) ) }
+				{ 0 < discount ? this.renderRow( translate( 'Your discount' ), -discount, 'discount', false, true ) : null }
```

Comment:
Hi! I've found a possible matching string that has already been translated **41** times:
`translate( 'Discount' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/19935-label-price-breakdown)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).